scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -1,131 +1,142 @@
+                  <span style="color: red;">{{ error }}</span><br>
                 {% endfor %}
-              </select>
+              </div>
             </div>
-          </div>
-          <div class="row mt-3 d-flex justify-content-center">
-            <div class="col-md-4 align-self-center">
-              {{ form.institutes.label(class="control-label",data_bs_toggle="tooltip", data_bs_placement="top", title="Allow case sharing only with preselected institutes.") }}
-              <select multiple="multiple" class="selectpicker" name="institutes" data-style="btn-secondary">
-              {% for choice in form.institutes.choices %}
-                <option value="{{choice[0]}}" {% if choice[0] in institute.get("collaborators", []) %} selected {% endif %}>{{choice[1]}}</option>
-              {% endfor %}
-              </select>
+            <div class="row mt-3 d-flex align-items-center justify-content-center">
+              <div class="col-7">
+                {{ form.pheno_group.label(class="control-label",data_bs_toggle="tooltip", data_bs_placement="top", title="Add a new phenotype group item to the list of phenotype shortcuts below.") }}
+                <input name="hpo_term" class="typeahead_hpo form-control" id="pheno_group" data-provide="typeahead" autocomplete="off" placeholder="Search HPO..">
+              </div>
+              <div class="d-block col-3">
+                {{ form.pheno_abbrev.label(class="control-label",data_bs_toggle="tooltip", data_bs_placement="top", title="A short name for this phenotype group.") }}
+                <input name="pheno_abbrev" class="form-control" name="pheno_abbrev" id="pheno_abbrev" placeholder="e.g.: NEUR" pattern=".{2,10}">
+              </div>
+              <div class="d-flex align-items-center col-2">
+                <button type="button" class="btn btn-secondary btn-small" onclick="validate_pheno()">Add custom phenotype group</button>
+              </div>
             </div>
-            {% if "admin" in current_user.roles %}
-              <!-- Admins can modify loqusdb settings -->
-              <div class="col-md-4">
-                {{ form.loqusdb_id.label(class="control-label", data_bs_toggle="tooltip", data_bs_placement="top", title="LoqudDB id" ) }}<br>
-                <select multiple class="selectpicker" name="loqusdb_id" data-style="btn-secondary">
-                  {% for instance in loqus_instances %}
-                    <option value="{{instance}}" {% if instance in institute.loqusdb_id %} selected {% endif %}>{{instance}}</option>
+
+            <div class="row mt-3 d-flex align-items-center">
+
+              <div class="col-4">
+                {{ form.gene_panels.label(class="control-label",data_bs_toggle="tooltip", data_bs_placement="top", title="Select gene panels that will be available for variants filtering.") }}
+                <select multiple="multiple" name="gene_panels" class="selectpicker ml-3">
```

Comment:
Ahem, *cough, cough* that style for dark mode.. 😉 
```suggestion
                <select multiple="multiple" name="gene_panels" class="selectpicker ml-3" data-style="btn-secondary">
```